src/app/fdctl/run/tiles/fd_bank.c

```diff
@@ -182,6 +183,12 @@ after_frag( void *             _ctx,
     ctx->metrics.txn_load_address_lookup_tables[ result ]++;
     if( FD_UNLIKELY( result!=FD_BANK_ABI_TXN_INIT_SUCCESS ) ) continue;
 
+    int precompile_result = fd_ext_bank_verify_precompiles( ctx->_bank, abi_txn );
+    if( FD_UNLIKELY( precompile_result ) ) {
+      FD_MCNT_INC( BANK_TILE, PRECOMPILE_VERIFY_FAILED, 1 );
```

Comment:
nit: could be a lot of cache traffic if someone spams txns that fail precompiles, they also don't pay right

Reply:
I think each precompile takes tens of microseconds of computation, so the cache traffic will be in the noise